fast/crud.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from sqlalchemy.orm import Session
+from db import models
+import schemas
+
+
+def get_all_author(
```

Comment:
Improvement: You can create a class with the argument `db`
This class can be used as a dependency and can be initialized with db in your routers.
With this approach, you won't need to pass db to every method

Reply:
Should I create this class in models.py?